hierarchicalforecast/evaluation.py

```diff
@@ -12,7 +12,12 @@
 
 # %% ../nbs/evaluation.ipynb 4
 class HierarchicalEvaluation:
-    
+    """Hierarchical Evaluation Class.
+    [Source code](https://github.com/dluuo/hierarchicalforecast/blob/main/hierarchicalforecast/evaluation.py).
+
```

Comment:
Let's add here the parameters of the constructor the `__init__` function. 
To make the library's documentation homogenous.